ppl/zqd/ztests/postgres/duplname.yaml

```diff
@@ -0,0 +1,15 @@
+tag: postgres
```

Comment:
How about creating a new `ppl/zqd/postgres/ztest` directory for all of these tests?